Source: http://flex.cvs.sourceforge.net/viewvc/flex/flex/tests/test-pthread/Makefile.am?r1=1.11&r2=1.12&view=patch
Upstream: in CVS
Reason: respect LDFLAGS when building the pthread test; we need this for -m32
        for multilib

--- flex-2.5.35/tests/test-pthread/Makefile.am	2006/11/09 20:14:49	1.11
+++ flex-2.5.35/tests/test-pthread/Makefile.am	2008/05/14 19:35:12	1.12
@@ -27,7 +27,7 @@
 OBJS = scanner.o # parser.o
 
 AM_CPPFLAGS = -I$(srcdir) -I$(top_srcdir) -I$(top_builddir)
-LDFLAGS = -lpthread
+LIBS = -lpthread
 #LFLAGS = --header="scanner.h"
 #YFLAGS = --defines --output=parser.c
 
@@ -37,7 +37,7 @@
 	$(FLEX) $(LFLAGS) $<
 
 $(testname)$(EXEEXT): $(OBJS)
-	$(CC) -o $@ $(LDFLAGS) $(OBJS) $(LOADLIBES)
+	$(CC) -o $@ $(LDFLAGS) $(OBJS) $(LIBS) $(LOADLIBES)
 
 test: $(testname)$(EXEEXT)
 	./$(testname) $(srcdir)/test-*.input
